.github/workflows/run-tests.yml

```diff
@@ -37,7 +41,8 @@ jobs:
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
           ref: ${{ env.COMMIT }}
-          submodules: true
+          # buf-breaking tries to compare HEAD against merge base so we need to be able to find it
+          fetch-depth: 100
```

Comment:
I know.. I was concerned that it might make this too slow. But maybe it's okay? As long as branches don't diverge too much 100 commits on main and 100 on the PR is probably enough to find the base, and if it can't find it, it still does the check against the head of main, which will find other issues.

I wish GHA could supply the merge base directly.. it obviously can compute it easily. But all the solutions I found online did something like this.